backend/src/main/java/com/crm/service/CarServiceImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.crm.service;
+
+import com.crm.dto.mapper.CarMapper;
+import com.crm.dto.response.CarResponse;
+import com.crm.model.db.CarEntity;
+import com.crm.repository.CarRepository;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CarServiceImpl implements CarService {
+
+    private final CarRepository carRepository;
+    private final CarMapper carMapper;
+
+    public CarServiceImpl(CarRepository carRepository, CarMapper carMapper) {
+        this.carRepository = carRepository;
+        this.carMapper = carMapper;
+    }
+
+    @Override
+    public Page<CarResponse> getCarsPaginated(int page, int size) {
+        Pageable pageable = PageRequest.of(page, size);
```

Comment:
Please move this directly to 37 line.

Reply:
I would leave as it is because variable pageable is used in 2 places in this method.